test/xpu_api/numerics/complex.number/cases.h

```diff
@@ -19,175 +19,175 @@
-    dpl::complex<double>(-0.5, -0.5),
-    dpl::complex<double>(-0., -0.5),
-    dpl::complex<double>(+0., -0.5),
-    dpl::complex<double>(0.5, -0.5),
-    dpl::complex<double>(1, -0.5),
-    dpl::complex<double>(2, -0.5),
-    dpl::complex<double>(INFINITY, -0.5),
-
-    dpl::complex<double>(NAN, -0.),
-    dpl::complex<double>(-INFINITY, -0.),
-    dpl::complex<double>(-2, -0.),
-    dpl::complex<double>(-1, -0.),
-    dpl::complex<double>(-0.5, -0.),
-    dpl::complex<double>(-0., -0.),
-    dpl::complex<double>(+0., -0.),
-    dpl::complex<double>(0.5, -0.),
-    dpl::complex<double>(1, -0.),
-    dpl::complex<double>(2, -0.),
-    dpl::complex<double>(INFINITY, -0.),
-
-    dpl::complex<double>(NAN, +0.),
-    dpl::complex<double>(-INFINITY, +0.),
-    dpl::complex<double>(-2, +0.),
-    dpl::complex<double>(-1, +0.),
-    dpl::complex<double>(-0.5, +0.),
-    dpl::complex<double>(-0., +0.),
-    dpl::complex<double>(+0., +0.),
-    dpl::complex<double>(0.5, +0.),
-    dpl::complex<double>(1, +0.),
-    dpl::complex<double>(2, +0.),
-    dpl::complex<double>(INFINITY, +0.),
-
-    dpl::complex<double>(NAN, 0.5),
-    dpl::complex<double>(-INFINITY, 0.5),
-    dpl::complex<double>(-2, 0.5),
-    dpl::complex<double>(-1, 0.5),
-    dpl::complex<double>(-0.5, 0.5),
-    dpl::complex<double>(-0., 0.5),
-    dpl::complex<double>(+0., 0.5),
-    dpl::complex<double>(0.5, 0.5),
-    dpl::complex<double>(1, 0.5),
-    dpl::complex<double>(2, 0.5),
-    dpl::complex<double>(INFINITY, 0.5),
-
-    dpl::complex<double>(NAN, 1),
-    dpl::complex<double>(-INFINITY, 1),
-    dpl::complex<double>(-2, 1),
-    dpl::complex<double>(-1, 1),
-    dpl::complex<double>(-0.5, 1),
-    dpl::complex<double>(-0., 1),
-    dpl::complex<double>(+0., 1),
-    dpl::complex<double>(0.5, 1),
-    dpl::complex<double>(1, 1),
-    dpl::complex<double>(2, 1),
-    dpl::complex<double>(INFINITY, 1),
-
-    dpl::complex<double>(NAN, 2),
-    dpl::complex<double>(-INFINITY, 2),
-    dpl::complex<double>(-2, 2),
-    dpl::complex<double>(-1, 2),
-    dpl::complex<double>(-0.5, 2),
-    dpl::complex<double>(-0., 2),
-    dpl::complex<double>(+0., 2),
-    dpl::complex<double>(0.5, 2),
-    dpl::complex<double>(1, 2),
-    dpl::complex<double>(2, 2),
-    dpl::complex<double>(INFINITY, 2),
-
-    dpl::complex<double>(NAN, INFINITY),
-    dpl::complex<double>(-INFINITY, INFINITY),
-    dpl::complex<double>(-2, INFINITY),
-    dpl::complex<double>(-1, INFINITY),
-    dpl::complex<double>(-0.5, INFINITY),
-    dpl::complex<double>(-0., INFINITY),
-    dpl::complex<double>(+0., INFINITY),
-    dpl::complex<double>(0.5, INFINITY),
-    dpl::complex<double>(1, INFINITY),
-    dpl::complex<double>(2, INFINITY),
-    dpl::complex<double>(INFINITY, INFINITY)
+    dpl::complex<double>( 1.e-6,  1.e-6),       // 0
+    dpl::complex<double>(-1.e-6,  1.e-6),       // 1
+    dpl::complex<double>(-1.e-6, -1.e-6),       // 2
+    dpl::complex<double>( 1.e-6, -1.e-6),       // 3
```

Comment:
Is this numbering important?
It is hard/inconvenient to maintain. What if some extra tests will be added in the middle?

Reply:
Changes has been removed from this file